src/lua/functions/core/game/bank_functions.hpp

```diff
@@ -8,7 +17,7 @@ class Bank;
 class BankFunctions final : LuaScriptInterface {
 	public:
 		static void init(lua_State* L) {
-			registerTable(L, "Bank");
+			registerSharedClass(L, "Bank", "");
```

Comment:
Did you test the bank? (i will, just wondering) the way this is done is by exposing static methods only, so the Table here was never actually an instance of anything. I'm not sure that we need to have it be a pointer of any kind

Reply:
I didn't test it, and I really did it because I have doubts about the correct modification in that part